src/test/suite/fixtures/recorded/languages/latex/changeCallee.yml

```diff
@@ -0,0 +1,23 @@
+languageId: latex
+command:
+  version: 1
+  spokenForm: change callee
+  action: clearAndSetSelection
+  targets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: functionCallee, includeSiblings: false}
+initialState:
+  documentContents: \href{https://some.url}{some text} \LaTeX
+  selections:
+    - anchor: {line: 0, character: 3}
+      active: {line: 0, character: 3}
+  marks: {}
+finalState:
+  documentContents: "{https://some.url}{some text} \\LaTeX"
```

Comment:
Genuine question here: do we want to keep the initial `\`?  Ie would there ever be a situation where we want to change the command and it wouldn't end up starting with a `\`?  Another way to look at it is: is the `\` considered part of the command name, or is it part of the overall call structure

Reply:
Let's leave this one for now